sample-cap/srv/orchestration/orchestration-service.ts

```diff
@@ -0,0 +1,30 @@
+import { Request as CdsRequest } from '@sap/cds';
```

Comment:
why using alias?

Reply:
Somehow my vscode won't resolve `req: Request` from CAP, but from node.
![image](https://github.com/user-attachments/assets/2926964f-d470-46e9-b5bb-357caba4b409)

But it also didn't help when using alias, so I changed it back.